examples/containers/.env

```diff
@@ -0,0 +1,3 @@
+SERVER_IMAGE=couchbase/server:7.0.2
```

Comment:
`COUCHBASE_SERVER_IMAGE` for consistency?

Reply:
Yes, do as I say not as I do